src/SFA.DAS.AssessorService.Api.Types/Consts/EmailTemplateNames.cs

```diff
@@ -4,5 +4,7 @@ public class EmailTemplateNames
     {
         public const string EPAOPrimaryContactAmended = "EPAOPrimaryContactAmended";
         public const string EPAOOrganisationDetailsAmended = "EPAOOrganisationDetailsAmended";
+        public const string APPLY_EPAO_INITIAL_SUBMISSION = "ApplyEPAOInitialSubmission";
```

Comment:
Is there some reason why you don't CamelCase the new const like the ones above?

Reply:
Cut and paste from Apply. Will change it.

Reply:
Done